File: pjecalc-fonte/negocio/br/jus/trt8/pjecalc/negocio/dominio/termo/Termo.java
/*
 * Decompiled with CFR 0.152.
 */
package br.jus.trt8.pjecalc.negocio.dominio.termo;

import br.jus.trt8.pjecalc.negocio.dominio.termo.ParametroDoTermo;
import java.io.Serializable;
import java.math.BigDecimal;

public interface Termo
extends Serializable {
    public BigDecimal resolverValor(ParametroDoTermo var1);
}

File: src/lib/pjecalc/core/dominio/termo/termo.ts
/**
 * Porte 1:1 de Termo.java (interface base, 14 linhas).
 * Contrato: toda classe que resolve um valor (Quantidade, Divisor, BaseTabelada, BaseVerba, etc.)
 * implementa `resolverValor(parametro): Decimal`.
 *
 * Ref: pjecalc-fonte/.../dominio/termo/Termo.java
 */
import type Decimal from 'decimal.js';
import type { ParametroDoTermo } from './parametro-do-termo';

export interface Termo {
  resolverValor(parametro: ParametroDoTermo): Decimal;
}
